src/recommenderapp/templates/landing_page.html

```diff
@@ -1,177 +1,94 @@
           <span class="navbar-toggler-icon"></span>
         </button>
-        <!-- <div class="collapse navbar-collapse" id="navbarSupportedContent">
-          <div class="d-flex">
-            <a class="nav-link" href="#">Link</a>
-          </div>
-        </div> -->
       </div>
     </nav>
-    <div class="heading1">
-      <div class="d-flex">
-        <h2>🎬Pick a Movie!🎬</h2>
-      </div>
-      <div class="d-flex">
-        <input
-          class="form-control mr-sm-2"
-          type="search"
-          placeholder="Search for a Movie"
-          aria-label="Search"
-          id="searchBox"
-        />
+    <div class="container" style="margin-top: 60px;">
+      <div class="heading1">
+        <!-- Heading for picking a movie -->
+        <h2><center>🎬 Pick a Movie! 🎬</center></h2>
       </div>
-    </div>
 
-    <!-- <div style="width: 600px; margin: auto">
-      <h2>Please select movies for training</h2>
-      <p style="width: 560px; margin: auto">
-        <label>Search Here</label>&nbsp;&nbsp;<input
-          type="text"
-          name="search"
-          id="searchBox"
-        />
-      </p>
-    </div> -->
-    <div style="width: 600px; margin: auto">
-      <h2>Selected movie :</h2>
-      <ul class="list-group" id="selectedMovies"></ul>
-    </div>
-    <div style="width: 600px; margin: auto">
-      <input
-        type="button"
-        class="btn btn-primary"
-        name="predict"
-        id="predict"
-        value="Predict"
-      />
-    </div>
-    <div style="width: 600px; margin: auto">
-      <h2>Predicted Movies</h2>
-      <form class="recos" id="recos">
-        <ul class="list-group" id="predictedMovies"></ul>
-      </form>
-    </div>
-    <div style="width: 600px; margin: auto">
-      <input
-        type="button"
-        class="btn btn-primary"
-        name="feedback"
-        data-toggle="modal"
-        id="feedback"
-        value="Give Feedback"
-        data-target="#exampleModalCenter"
-      />
-      <!--TODO Button trigger modal-->
-      <button
-        type="button"
-        data-bs-toggle="modal"
-        data-bs-target="#exampleModalCenter"
-      >
-        button
-      </button>
-      <div
-        class="modal fade"
-        id="exampleModalCenter"
-        tabindex="-1"
-        role="dialog"
-        aria-labelledby="exampleModalCenterTitle"
-        aria-hidden="true"
-      >
-        <div class="modal-dialog modal-dialog-centered" role="document">
-          <div class="modal-content">
-            <div class="modal-header">
-              <h5 class="modal-title" id="exampleModalLongTitle">
-                Modal title
-              </h5>
-              <button
-                type="button"
-                class="close"
-                data-bs-dismiss="modal"
-                aria-label="Close"
-              >
-                <span aria-hidden="true">&times;</span>
-              </button>
+
+      <div class="row" style="margin-top: 25px;">
```

Comment:
Try using rem instead of px, instead of row, col (Table) use flexBox.

Don't add styles in HTML, define a class and move it to stylesheet.css